### ISSUE TITLE
ConBee II Stick has the status "Joining ..." (Newly purchased / Latest firmware)

### ISSUE THREAD

**Issue by kkillius** [dresden-elektronik/deconz-rest-plugin#3429]
## Describe the bug

My newly purchased ConBee II Stick does not run on any platform. My main test runs on a Raspberry 3. I also completely reinstalled it. I completely followed the instructions below:
- https://phoscon.de/de/conbee2/install#raspbian
- https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually#update-in-raspbian

## Expected behavior

ConBee II Stick has the status "Joining ..."

## Screenshots

<img width="1352" alt="Bildschirmfoto 2020-10-16 um 23 33 48" src="https://user-images.githubusercontent.com/10324559/96311199-4008ca00-1009-11eb-8b97-be55a221f0f9.png">

## Environment

 - Host system: Raspberry Pi 3
 - Running method: Raspbian
 - Firmware version: 26660700
 - deCONZ version: 2.05.85
 - Device: ConBee II
 - Do you use an USB extension cable: yes

### deCONZ Logs

pi@WETRIX-PI:~ $ deCONZ --dbg-error=2 --dbg-info=2 --dbg-ota=0 --dbg-zdp=0 --dbg-zcl=0 --db-aps=0 --dbg-http=0
QCoreApplication::arguments: Please instantiate the QApplication object first
libEGL warning: DRI2: failed to authenticate
qt5ct: using qt5ct plugin
qt5ct: D-Bus global menu: no
libpng warning: iCCP: known incorrect sRGB profile
23:19:45:417 HTTP Server listen on address 0.0.0.0, port: 80, root: /usr/share/deCONZ/webapp/
23:19:45:473 CTRL. 3.27.223:19:45:790 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
23:19:45:790 dev /dev/ttyAMA0 (/dev/ttyAMA0)
23:19:45:790 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
23:19:45:791 ZCLDB init file /home/pi/.local/share/dresden-elektronik/deCONZ/zcldb.txt
23:19:46:171 parent process bash
23:19:46:171 gw run mode: normal
23:19:46:171 GW sd-card image version file does not exist: /home/pi/.local/share/dresden-elektronik/deCONZ/gw-version
23:19:46:172 sd-card cid: 1b534d303030303010ffffffff00f919
23:19:46:174 DB sqlite version 3.27.2
23:19:46:175 DB PRAGMA page_count: 33
23:19:46:175 DB PRAGMA page_size: 4096
23:19:46:175 DB PRAGMA freelist_count: 0
23:19:46:175 DB file size 135168 bytes, free pages 0
23:19:46:176 DB PRAGMA user_version: 7
23:19:46:176 DB cleanup
23:19:46:176 DB create temporary views
23:19:46:177 DB view [0] created
23:19:46:178 DB view [1] created
23:19:46:178 DB view [2] created
23:19:46:179 DB view [3] created
23:19:46:179 sql exec SELECT apikey,devicetype,createdate,lastusedate,useragent FROM auth
23:19:46:180 sql exec SELECT key FROM config2
23:19:46:180 sql exec SELECT key,value FROM config2
23:19:46:180 Load config UTC: 2020-10-16T21:11:59 from db.
23:19:46:180 Load config announceinterval: 45 from db.
23:19:46:180 Load config announceurl: https://phoscon.de/discover from db.
23:19:46:180 Load config apiversion: 2.05.85 from db.
23:19:46:181 Load config bridgeid: 00212EFFFF05FA1A from db.
23:19:46:181 Load config datastoreversion: 93 from db.
23:19:46:181 Load config dhcp: true from db.
23:19:46:181 Load config disablePermitJoinAutoOff: false from db.
23:19:46:181 Load config discovery: false from db.
23:19:46:181 Load config factorynew: false from db.
23:19:46:181 Load config fwneedupdate: false from db.
23:19:46:181 Load config fwupdatestate: idle from db.
23:19:46:181 Load config fwversion: 0x26660700 from db.
23:19:46:181 Load config gateway: 127.0.0.1 from db.
23:19:46:181 Load config group0: 65520 from db.
23:19:46:181 Load config groupdelay: 50 from db.
23:19:46:181 Load config gwpassword: $1$8282jdkm$hG3WKnMkTS8aJ6c9DoPg.0 from db.
23:19:46:181 Load config gwusername: delight from db.
23:19:46:181 Load config homebridge: not-managed from db.
23:19:46:181 Load config homebridge-pin:  from db.
23:19:46:181 Load config homebridgepin:  from db.
23:19:46:181 Load config homebridgeupdate: false from db.
23:19:46:181 Load config homebridgeupdateversion:  from db.
23:19:46:181 Load config homebridgeversion:  from db.
23:19:46:181 Load config ipaddress: 192.168.178.33 from db.
23:19:46:181 Load config linkbutton: false from db.
23:19:46:181 Load config localtime: 2020-10-16T23:11:59 from db.
23:19:46:181 Load config mac: 38:60:77:7c:53:18 from db.
23:19:46:181 Load config modelid: deCONZ from db.
23:19:46:181 Load config name: Phoscon-GW from db.
23:19:46:181 Load config netmask: 255.0.0.0 from db.
23:19:46:181 Load config networkopenduration: 180 from db.
23:19:46:182 Load config otauactive: false from db.
23:19:46:182 Load config otaustate: off from db.
23:19:46:182 Load config panid: 0 from db.
23:19:46:182 Load config permitjoin: 0 from db.
23:19:46:182 Load config permitjoinfull: 0 from db.
23:19:46:182 Load config port: 80 from db.
23:19:46:182 Load config portalservices: false from db.
23:19:46:182 Load config proxyaddress: none from db.
23:19:46:182 Load config proxyport: 0 from db.
23:19:46:182 Load config rfconnect: 1 from db.
23:19:46:182 Load config rfconnected: false from db.
23:19:46:182 Load config rgbwdisplay: 1 from db.
23:19:46:182 Load config runmode: normal from db.
23:19:46:182 Load config starterkitid:  from db.
23:19:46:182 Load config swcommit: 0608e1538c89f991fb5186ca2035580ed8cb7c8b from db.
23:19:46:182 Load config swupdatestate: allreadytoinstall from db.
23:19:46:182 Load config swversion: 2.05.85 from db.
23:19:46:182 Load config system: linux-gw from db.
23:19:46:182 Load config timeformat: 24h from db.
23:19:46:182 Load config timezone: Etc/GMT from db.
23:19:46:182 Load config updatechannel: stable from db.
23:19:46:182 Load config uuid: 1244591a-eab0-4603-81b5-53a19d56834e from db.
23:19:46:182 Load config websocketnotifyall: true from db.
23:19:46:182 Load config websocketport: 443 from db.
23:19:46:182 Load config wifi: not-configured from db.
23:19:46:182 Load config wifibackupname:  from db.
23:19:46:182 Load config wifibackuppw:  from db.
23:19:46:182 Load config wifibackuppwenc:  from db.
23:19:46:183 Load config wifichannel: 1 from db.
23:19:46:183 Load config wificlientname:  from db.
23:19:46:183 Load config wificlientpw:  from db.
23:19:46:183 Load config wifiip: 192.168.8.1 from db.
23:19:46:183 Load config wifilastupdated: 0 from db.
23:19:46:183 Load config wifimgmt: 0 from db.
23:19:46:183 Load config wifiname:  from db.
23:19:46:183 Load config wifipageactive: false from db.
23:19:46:183 Load config wifipw:  from db.
23:19:46:183 Load config wifipwenc:  from db.
23:19:46:183 Load config wifitype: accesspoint from db.
23:19:46:183 Load config workingname:  from db.
23:19:46:183 Load config workingpw:  from db.
23:19:46:183 Load config workingpwenc:  from db.
23:19:46:183 Load config workingtype:  from db.
23:19:46:183 Load config zclvaluemaxage: 0 from db.
23:19:46:183 Load config zigbeechannel: 11 from db.
23:19:46:183 sql exec SELECT key,value FROM userparameter
23:19:46:184 sql exec SELECT * FROM groups
23:19:46:184 Sqlite group: gid = 0xFFF0
23:19:46:184 Sqlite group: name = All
23:19:46:184 Sqlite group: state = normal
23:19:46:184 Sqlite group: hidden = false
23:19:46:184 Sqlite group: type = LightGroup
23:19:46:184 Sqlite group: class = Other
23:19:46:185 DB found group All 0xFFF0
23:19:46:185 ~Resource() /groups 0x7e9b3ab0
23:19:46:185 sql exec SELECT * FROM resourcelinks
23:19:46:185 sql exec SELECT * FROM scenes
23:19:46:185 sql exec SELECT * FROM rules
23:19:46:186 sql exec SELECT * FROM schedules
23:19:46:186 sql exec SELECT * FROM sensors
23:19:46:186 Sqlite sensors: sid = 1
23:19:46:186 Sqlite sensors: name = Daylight
23:19:46:186 Sqlite sensors: type = Daylight
23:19:46:187 Sqlite sensors: modelid = PHDL00
23:19:46:187 Sqlite sensors: manufacturername = Philips
23:19:46:187 Sqlite sensors: uniqueid = 00:21:2e:ff:ff:05:fa:1a-01
23:19:46:187 Sqlite sensors: swversion = 1.0
23:19:46:187 Sqlite sensors: state = {"dark":true,"daylight":false,"lastupdated":"2020-10-16T21:13:58.728","status":230,"sunrise":"2020-10-16T06:00:24","sunset":"2020-10-16T16:39:46"}
23:19:46:187 Sqlite sensors: config = {"configured":true,"lat":"51.3553","long":"6.6366","on":true,"sunriseoffset":30,"sunsetoffset":-30}
23:19:46:187 Sqlite sensors: deletedState = normal
23:19:46:187 Sqlite sensors: mode = 1
23:19:46:187 DB found sensor Daylight 1
23:19:46:188 ~Resource() /sensors 0x7e9b3ac8
23:19:46:189 sql exec SELECT * FROM gateways
21:19:46:190 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:19:46:191 started websocket server at port 443
21:19:46:192 <?xml version="1.0" encoding="UTF-8" ?>
21:19:46:192 <root xmlns="urn:schemas-upnp-org:device-1-0">
21:19:46:192   <specVersion>
21:19:46:192     <major>1</major>
21:19:46:192     <minor>0</minor>
21:19:46:192   </specVersion>
21:19:46:192   <URLBase>http://192.168.178.33:80/</URLBase>
21:19:46:192   <device>
21:19:46:192     <deviceType>urn:schemas-upnp-org:device:Basic:1</deviceType>
21:19:46:192     <friendlyName>Phoscon-GW (192.168.178.33)</friendlyName>
21:19:46:193     <manufacturer>Royal Philips Electronics</manufacturer>
21:19:46:193     <manufacturerURL>http://www.dresden-elektronik.de</manufacturerURL>
21:19:46:193     <modelDescription>Philips hue compatible Personal Wireless Lighting</modelDescription>
21:19:46:193     <modelName>Philips hue bridge 2015</modelName>
21:19:46:193     <modelNumber>BSB002</modelNumber>
21:19:46:193     <modelURL>http://www.dresden-elektronik.de</modelURL>
21:19:46:193     <serialNumber>00212E05FA1A</serialNumber>
21:19:46:193     <UDN>uuid:1244591a-eab0-4603-81b5-53a19d56834e</UDN>
21:19:46:193     <presentationURL>index.html</presentationURL>
21:19:46:194     <iconList>
21:19:46:194       <icon>
21:19:46:194         <mimetype>image/png</mimetype>
21:19:46:194         <height>48</height>
21:19:46:194         <width>48</width>
21:19:46:194         <depth>24</depth>
21:19:46:194         <url>hue_logo_0.png</url>
21:19:46:194       </icon>
21:19:46:194     </iconList>
21:19:46:194   </device>
21:19:46:194 </root>
21:19:46:194 <?xml version="1.0" encoding="UTF-8" ?>
21:19:46:194 <root xmlns="urn:schemas-upnp-org:device-1-0">
21:19:46:194   <specVersion>
21:19:46:195     <major>1</major>
21:19:46:195     <minor>0</minor>
21:19:46:195   </specVersion>
21:19:46:195   <URLBase>http://192.168.178.33:80/</URLBase>
21:19:46:195   <device>
21:19:46:195     <deviceType>urn:schemas-upnp-org:device:Basic:1</deviceType>
21:19:46:195     <friendlyName>Phoscon-GW (192.168.178.33)</friendlyName>
21:19:46:195     <manufacturer>Royal Philips Electronics</manufacturer>
21:19:46:195     <manufacturerURL>http://www.dresden-elektronik.de</manufacturerURL>
21:19:46:195     <modelDescription>Philips hue compatible Personal Wireless Lighting</modelDescription>
21:19:46:195     <modelName>Philips hue bridge 2015</modelName>
21:19:46:195     <modelNumber>BSB002</modelNumber>
21:19:46:195     <modelURL>http://www.dresden-elektronik.de</modelURL>
21:19:46:195     <serialNumber>00212E05FA1A</serialNumber>
21:19:46:195     <UDN>uuid:1244591a-eab0-4603-81b5-53a19d56834e</UDN>
21:19:46:195     <presentationURL>index.html</presentationURL>
21:19:46:195     <iconList>
21:19:46:195       <icon>
21:19:46:195         <mimetype>image/png</mimetype>
21:19:46:195         <height>48</height>
21:19:46:195         <width>48</width>
21:19:46:196         <depth>24</depth>
21:19:46:196         <url>hue_logo_0.png</url>
21:19:46:196       </icon>
21:19:46:196     </iconList>
21:19:46:196   </device>
21:19:46:196 </root>
21:19:46:198 [INFO] - Found file containing button maps. Parsing data...
21:19:46:202 [ERROR] - Button map 'sunricherCCTMap' in JSON file has no assigned ModelIDs. Skip loading button map...
21:19:46:217 [INFO] - Button maps loaded.
21:19:46:220 dlg action: Read binding table
21:19:46:220 found node plugin: libde_rest_plugin.so - REST API Plugin
21:19:46:228 found node plugin: libde_signal_plugin.so - Signal Monitor Plugin
21:19:46:231 Otau image path does not exist: /home/pi/otau
21:19:46:294 found node plugin: libstd_otau_plugin.so - STD OTAU Plugin
21:19:46:353 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:19:46:354 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:19:46:354 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:19:46:704 discard sensor state push for 1: state/dark (already pushed)
21:19:46:708 discard sensor state push for 1: state/status (already pushed)
21:19:46:710 discard sensor state push for 1: state/lastupdated (already pushed)
21:19:47:245 UPNP socket not bound, state: 0
21:19:47:410 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:19:47:410 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:19:47:410 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:19:47:411 auto connect com /dev/ttyAMA0
21:19:47:415 COM check bootloader
21:19:47:434 API error 1, /, unauthorized user
21:19:48:088 New websocket 192.168.178.59:52290 (state: 3) 
21:19:48:513 New websocket 192.168.178.59:52292 (state: 3) 
21:19:49:418 COM timout query bootloader, assume application
21:19:49:418 Serial com connected
21:19:50:845 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:19:50:845 device state timeout ignored in state 2
21:19:51:315 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:19:51:315 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:19:51:315 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:19:51:316 GW update firmware not found: 
21:19:51:505 Announced to internet https://phoscon.de/discover
21:19:51:506 discovery server date: Fri, 16 Oct 2020 21:19:51 GMT
21:19:51:506 	 local time seems to be ok
21:19:51:506 discovery found version 2.04.35 for update channel stable
21:19:52:167 API error 1, /, unauthorized user
21:19:53:758 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:19:53:758 device state timeout ignored in state 2
21:19:56:259 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:19:56:758 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:19:56:758 device state timeout ignored in state 2
21:19:57:470 API error 1, /, unauthorized user
21:19:59:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:19:59:759 device state timeout ignored in state 2
21:20:01:312 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:01:312 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:01:312 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:01:313 GW update firmware not found: 
21:20:02:470 API error 1, /, unauthorized user
21:20:02:758 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:02:758 device state timeout ignored in state 2
21:20:05:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:05:759 device state timeout ignored in state 2
21:20:06:259 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:20:06:259 networkState: CC_ReconnectNetwork
21:20:06:259 start reconnect to network
21:20:06:971 API error 1, /, unauthorized user
21:20:08:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:08:760 device state timeout ignored in state 2
21:20:11:259 try to reconnect to network try=1
21:20:11:359 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:11:359 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:11:359 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:11:359 GW update firmware not found: 
21:20:11:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:11:759 device state timeout ignored in state 2
21:20:12:167 API error 1, /, unauthorized user
21:20:14:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:14:759 device state timeout ignored in state 2
21:20:16:259 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:20:16:259 try to reconnect to network try=2
21:20:17:517 API error 1, /, unauthorized user
21:20:17:758 command queue EVENT_TIMEOUT, cmd: CMD_CHANGE_NET_STATE, seq: 8
21:20:20:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:20:759 device state timeout ignored in state 2
21:20:21:258 try to reconnect to network try=3
21:20:21:343 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:21:343 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:21:343 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:21:344 GW update firmware not found: 
21:20:22:007 API error 1, /, unauthorized user
21:20:22:759 command queue EVENT_TIMEOUT, cmd: CMD_CHANGE_NET_STATE, seq: 8
21:20:25:759 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:25:759 device state timeout ignored in state 2
21:20:26:259 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:20:26:259 try to reconnect to network try=4
21:20:27:473 API error 1, /, unauthorized user
21:20:27:758 command queue EVENT_TIMEOUT, cmd: CMD_CHANGE_NET_STATE, seq: 8
21:20:30:758 void zmMaster::handleStateStatus(zmMaster::MasterEvent) EVENT_TIMEOUT
21:20:30:758 device state timeout (handled)
21:20:30:758 MASTER kill cmd 0x08 (ERROR)
21:20:30:759 MASTER kill cmd 0x08 (ERROR)
21:20:30:759 MASTER kill cmd 0x08 (ERROR)
21:20:30:759 MASTER kill cmd 0x08 (ERROR)
21:20:30:764 Serial com disconnected, reason: 1
21:20:30:765 device disconnected reason: 1, index: 0
21:20:31:259 failed to reconnect to network try=5
21:20:31:259 wait reconnect 15 seconds
21:20:31:314 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:31:314 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:31:314 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:31:314 GW firmware update select USB device
21:20:31:314 GW update firmware not found: 
21:20:32:258 void zmMaster::handleStateIdle(zmMaster::MasterEvent) not connected goto OFF state
21:20:32:258 device state timeout ignored in state 1
21:20:32:259 wait reconnect 14 seconds
21:20:32:480 API error 1, /, unauthorized user
21:20:33:259 wait reconnect 13 seconds
21:20:34:259 wait reconnect 12 seconds
21:20:35:259 wait reconnect 11 seconds
21:20:36:261 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:20:36:261 failed to reconnect to network try=6
21:20:36:261 wait reconnect 10 seconds
21:20:36:970 API error 1, /, unauthorized user
21:20:37:259 wait reconnect 9 seconds
21:20:38:259 wait reconnect 8 seconds
21:20:39:259 wait reconnect 7 seconds
21:20:40:259 wait reconnect 6 seconds
21:20:41:259 failed to reconnect to network try=7
21:20:41:260 wait reconnect 5 seconds
21:20:41:381 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:41:381 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:41:381 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:41:381 GW firmware update select USB device
21:20:41:381 GW update firmware not found: 
21:20:42:259 wait reconnect 4 seconds
21:20:42:529 API error 1, /, unauthorized user
21:20:43:259 wait reconnect 3 seconds
21:20:44:259 wait reconnect 2 seconds
21:20:45:260 wait reconnect 1 seconds
21:20:45:349 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:45:349 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:45:349 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:45:349 auto connect com /dev/ttyACM0
21:20:45:365 COM check bootloader
21:20:45:379 COM detected application
21:20:45:379 Serial com connected
21:20:45:380 could not delete /var/tmp/deconz-firmware-version
21:20:45:384 Master: read param with arg 0x18
21:20:45:385 Master: read param with arg 0x13
21:20:45:385 Master: read param with arg 0x13
21:20:45:386 Master: read param with arg 0x18
21:20:45:386 Master: read param with arg 0x13
21:20:45:386 Master: read param with arg 0x13
21:20:45:617 Device firmware version 0x26660700 ConBee II
21:20:45:621 unlocked max nodes: 200
21:20:45:621 has feature: max nodes (0x11), status: 0x00
21:20:45:626 has feature: std security (0x01), status: 0x04
21:20:45:636 has feature: link security (0x02), status: 0x00
21:20:45:641 has feature: high security (0x03), status: 0x04
21:20:45:649 has feature: ethernet (0x04), status: 0x04
21:20:45:654 has feature: mass storage device (MSD) (0x05), status: 0x04
21:20:45:659 has feature: device firmware upgrade (DFU) (0x06), status: 0x04
21:20:45:664 has feature: ZigBee Light Link (ZLL) (0x07), status: 0x04
21:20:45:669 has feature: interpan (0x08), status: 0x04
21:20:45:674 has feature: button 1 (0x09), status: 0x04
21:20:45:680 has feature: button 1 (0x0A), status: 0x04
21:20:45:685 has feature: LED 1 (0x0B), status: 0x00
21:20:45:690 has feature: LED 2 (0x0C), status: 0x00
21:20:45:695 has feature: LED 3 (0x0D), status: 0x04
21:20:45:700 has feature: LED 4 (0x0E), status: 0x04
21:20:45:705 has feature: LED 5 (0x0F), status: 0x04
21:20:45:711 has feature: LED RGB (0x10), status: 0x04
21:20:45:715 has feature: max nodes (0x11), status: 0x04
21:20:45:721 Device protocol version: 0x010C
21:20:45:759 void deCONZ::zmNode::setFetched(deCONZ::RequestId, bool) fetched item: 2, node: 0x0000
21:20:45:759 new node - ext: 0x00212effff05fa1a, nwk: 0x0000
21:20:45:760 void deCONZ::zmNode::setFetched(deCONZ::RequestId, bool) fetched item: 2, node: 0x0000
21:20:45:841 void deCONZ::zmNode::setFetched(deCONZ::RequestId, bool) fetched item: 1, node: 0x0000
21:20:45:866 Master: read param with arg 0x19
21:20:45:895 Current channel 11
21:20:45:944 CTRL got nwk update id 0
21:20:45:969 CTRL ANT_CTRL 0x03
21:20:46:029 Device protocol version: 0x010C
21:20:46:051 void deCONZ::zmNode::setFetched(deCONZ::RequestId, bool) fetched item: 1, node: 0x0000
21:20:46:066 Master: read param with arg 0x19
21:20:46:119 Current channel 11
21:20:46:148 CTRL got nwk update id 0
21:20:46:161 CTRL ANT_CTRL 0x03
21:20:46:260 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:20:47:424 API error 1, /, unauthorized user
21:20:51:015 DB save zll database items 0x00000884
21:20:51:015 DB sql exec REPLACE INTO auth (apikey, devicetype, createdate, lastusedate, useragent) VALUES ('5159C37F23', 'Phoscon#B1024x1169', '2020-10-16T21:13:22', '2020-10-16T21:20:47', 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.75 Safari/537.36')
21:20:51:018 DB sql exec REPLACE INTO sensors (sid, name, type, modelid, manufacturername, uniqueid, swversion, state, config, fingerprint, deletedState, mode) VALUES ('1', 'Daylight', 'Daylight', 'PHDL00', 'Philips', '00:21:2e:ff:ff:05:fa:1a-01', '1.0', '{"dark":true,"daylight":false,"lastupdated":"2020-10-16T21:19:46.190","status":230,"sunrise":"2020-10-16T06:00:24","sunset":"2020-10-16T16:39:46"}', '{"configured":true,"lat":"51.3553","long":"6.6366","on":true,"sunriseoffset":30,"sunsetoffset":-30}', '', 'normal', '1')
21:20:51:019 DB sql exec UPDATE devices SET nwk = 0 WHERE mac = '00:21:2e:ff:ff:05:fa:1a';INSERT INTO devices (mac,nwk,timestamp) SELECT '00:21:2e:ff:ff:05:fa:1a', 0, strftime('%s','now') WHERE (SELECT changes() = 0);
21:20:51:074 DB saved in 58 ms
21:20:51:315 COM: /dev/ttyAMA0 :  (0x0000/0x0000)
21:20:51:315 dev /dev/ttyAMA0 (/dev/ttyAMA0)
21:20:51:315 COM: /dev/ttyACM0 : ConBee II (0x1CF1/0x0030)
21:20:51:316 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26660700.bin.GCF
21:20:51:316 GW firmware version: 0x26660700
21:20:51:316 GW firmware version is up to date: 0x26660700
21:20:52:215 API error 1, /, unauthorized user
21:20:56:259 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:20:57:435 API error 1, /, unauthorized user
21:21:02:464 API error 1, /, unauthorized user
21:21:06:259 Daylight now: nightStart, status: 230, daylight: 0, dark: 1
21:21:06:962 API error 1, /, unauthorized user
21:21:12:217 API error 1, /, unauthorized user

**Comment by Mimiix** [comment]
Hi @kkillius 
It seems that it is not detected:
21:20:32:258 void zmMaster::handleStateIdle(zmMaster::MasterEvent) **not connected** goto OFF state

Also, What OS is this? Just raspbian or something else?  What Extension cable are you using? Are the other devices attached?

**Comment by manup** [comment]
If the stick doesn't manage to create the network and move to status "In Network" it can be an indication that the Zigbee channel is too occupied or some heavy interference is happening.

Could you please try to change the Zigbee channel from within the Phoscon App
Also do you have any other USB devices connected to the Raspberry Pi (SSD or Bluetooth devices)?

**Comment by kkillius** [comment]
@Mimiix :
> Also, What OS is this? Just raspbian or something else? 
I tried Raspian, Debian and Windows 10
> What Extension cable are you using?
0,5m and 3,0m
> Are the other devices attached?
No no other Devices and no Wifi and Bluetooth are disabled. 

@manup :
> Could you please try to change the Zigbee channel from within the Phoscon App
I couldn't change a setting. He couldn't save any changes to the stick.
> Also do you have any other USB devices connected to the Raspberry Pi (SSD or Bluetooth devices)?
No nothing.

@Mimiix and @manup :
I ordered a new stick from Amazon and sent the other one back.
With the new stick there were no problems straight away and the status "In Network" was displayed.
So it was probably a hardware defect.

Thanks for your help anyway.
